src/Common/src/System/SR.cs

```diff
@@ -89,3 +97,20 @@ internal static string Format(string resourceFormat, object p1, object p2, objec
         }
     }
 }
+
+namespace System.Resources.Diagnostics
+{
+    [AttributeUsage(AttributeTargets.All)]
+    [ExcludeFromCodeCoverage]
+    internal sealed class ExcludeFromCodeCoverageAttribute : Attribute
```

Comment:
I don't feel very strongly about this, but maybe we should use a namespace that would be more obvious code smell if anyone but this file imported it. e.g. Internal.ExcludeSRFromCodeCoverge or something like that...